doc/man3/SSL_CIPHER_get_name.pod

```diff
@@ -88,6 +92,13 @@ TLS 1.3 cipher suites) B<NID_auth_any> is returned. Examples (not comprehensive)
 SSL_CIPHER_is_aead() returns 1 if the cipher B<c> is AEAD (e.g. GCM or
 ChaCha20/Poly1305), and 0 if it is not AEAD.
 
+SSL_CIPHER_find() returns a B<SSL_CIPHER> structure which has the cipher ID B<ptr>.
```

Comment:
It's also unclear to me how they could get that information. But I do think we have an API that returns that, so maybe a reference to that should get added?